xray_vision/qt_widgets/__init__.py

```diff
@@ -100,6 +101,17 @@ def __init__(self, data_list, key_list,
         # trigger the image to draw
         self._messenger.sl_update_image(0)
 
+        self.key_list = key_list
```

Comment:
Is this not performed at the [messenger level](https://github.com/Nikea/xray-vision/pull/85/files#diff-99687c92a129682ef31b564362422319R82)?